package.json

```diff
@@ -19,6 +19,8 @@
     "lint:ci": "npm run lint && npm run lint:styles",
     "pree2e": "webdriver-manager update --standalone false --gecko false",
     "e2e": "ng e2e",
+    "docs": "./node_modules/.bin/compodoc -p src/tsconfig.app.json -d docs",
```

Comment:
Do we really need to specify the relative path to the `.bin` folder? I believe it could be done in a similar to other commands manner when an executable is resolved by npm (https://github.com/akveo/ngx-admin/pull/1327/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R6).

Reply:
Hi @nnixaa ,
This is how they use it as stated in the documentation
https://compodoc.github.io/website/guides/installation.html

Reply:
Yeah I see, but this doesn't seem to be right, as we hardcode path to .bin folder. Could you try changing it to:

```
"compodoc":  "compodoc",
"docs": "compodoc -p src/tsconfig.app.json -d docs",
"docs:serve": "compodoc -p src/tsconfig.app.json -d docs -s",
```